tests/firefox/prefs/search_suggestions_address_bar_enabled_disabled.py

```diff
@@ -0,0 +1,93 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from targets.firefox.fx_testcase import *
+
+
+class Test(FirefoxTest):
+
+    @pytest.mark.details(
+        description='Search suggestions in address bar can be successfully enabled/disabled',
+        test_case_id='143593',
+        test_suite_id='2241',
+        locale=['en-US'],
+    )
+    def run(self, firefox):
+        preferences_search_page = AboutPreferences.ABOUT_PREFERENCE_SEARCH_PAGE_PATTERN
+        suggestions_displayed_pattern = Pattern('suggestions_displayed.png')
+        browsing_history_search_bar_pattern = Pattern('browsing_history_search_bar.png')
+        show_suggestions_unchecked_greyed_pattern = Pattern('show_suggestions_unchecked_greyed.png')
```

Comment:
This pattern don't guarantee proper recognition of check-boxes in it because of large size